Socket/src/main/java/Server.java

```diff
@@ -0,0 +1,65 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Server {
+    public static void main(String[] args) {
+        try {
+            ServerSocket serverSocket = new ServerSocket(8081);
+            System.out.println("Server is ready, port 8081");
+
+            Socket clientSocket = serverSocket.accept();
+            System.out.println("Client connected: " + clientSocket.getInetAddress());
+
+            BufferedReader in = new BufferedReader(new InputStreamReader(clientSocket.getInputStream()));
+            PrintWriter out = new PrintWriter(clientSocket.getOutputStream(), true);
+
+            out.println("Hello");
+            String response = in.readLine();
+            System.out.println("Client: " + response);
+
+            boolean russianLetters = false;
+            for (char ru : response.toCharArray()) {
+                {
+                    russianLetters = true;
+                    break;
+                }
+            }
```

Comment:
according to this for loop Russian letters will always be present, and need to be redone